spec/models/comment_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'rails_helper'
+
+describe Comment, type: :model do
+
+  subject { Comment.new(content: "I'm a comment :)") }
+  
+  describe "Validations" do
+
+    describe '#content' do
+      it 'is invalid when blank' do
```

Comment:
Can you test a positive scenario as well? That comment with content is valid?

Reply:
Added 😅